libraries/wallet/wallet.cpp

```diff
@@ -1159,10 +1159,7 @@ class wallet_api_impl
         FC_THROW("No asset with that symbol exists!");
       optional<account_id_type> new_issuer_account_id;
       if (new_issuer)
-      {
-        account_object new_issuer_account = get_account(*new_issuer);
-        new_issuer_account_id = new_issuer_account.id;
-      }
+        FC_THROW("The use of 'new_issuer' is no longer supported. Please use `update_asset_issuer' instead!");
```

Comment:
This change need to be done only after the hard fork, otherwise people will be unable to update issuer before the hard fork with this command via this cli_wallet. Perhaps not a big deal though.